Src/BootCamp.Chapter/Checks.cs

```diff
@@ -17,25 +17,25 @@ public static class Checks
         public static int PromptInt(string message)
         {
             // To do: call your implementation. 
```

Comment:
Please remove redundant comments. You know what you do and why the comments are no longer needed.